internal/security/secretstore/init.go

```diff
@@ -435,6 +436,27 @@ func (b *Bootstrap) BootstrapHandler(ctx context.Context, _ *sync.WaitGroup, _ s
 		}
 	}
 
+	// Configure Kong Admin API
+	lc.Info("Starting the Kong Admin API config file creation")
+
+	// Get an instance of KongAdminAPI and map the paths from configuration.toml
+	ka := NewKongAdminAPI()
+	ka.paths.config = kongAdminConfig.ConfigFilePath
+	ka.paths.template = kongAdminConfig.ConfigTemplatePath
+	ka.paths.jwt = kongAdminConfig.ConfigJWTPath
+
+	// Setup Kong Admin API loopback configuration
+	//
+	// For context - this process doesn't actually talk to Kong, it creates the configuration
+	// file and JWT necessary in order for the Kong process to bootstrap itself with a properly
+	// locked down Admin API and enable security-proxy-setup with the JWT in order to setup
+	// the services/routes as configured.
```

Comment:
Add addition `why` comment for why this code is in Secret Store Setup when it has nothing to do with setting up the Secret Store.

Reply:
Done.